Wireframe/my_work.html

```diff
@@ -20,19 +20,19 @@ <h2>Purpose of a Wireframe</h2>
                     A wireframe is a visual diagram that outlines the structure and layout of a website, app, or other digital product. The purpose of a wireframe is to:
                 </p>
                 <ul>
-                    <li><b>Plan the Layout:</b><br>
+                    <li><strong>Plan the Layout:</strong><br>
                         Wireframes help designers and developers plan the placement of elements like buttons, images, and text.
                     </li>
-                    <li><b>Communicate functionality:</b><br>
+                    <li><strong>Communicate functionality:</strong><br>
                         Wireframes show how users will interact with the product and how it's expected to work.
                     </li>
-                    <li><b>Reduce trial and error:</b><br>
+                    <li><strong>Reduce trial and error:</strong><br>
                         Wireframes help teams work out issues before investing too much time or money into the project. 
                     </li>
-                    <li><b>Explore ideas:</b><br>
+                    <li><strong>Explore ideas:</strong><br>
                         Wireframes allow teams to explore different design approaches and aesthetics without writing code. 
                     </li>
-                    <li><b>Get feedback:</b><br>
+                    <li><strong>Get feedback:</strong><br>
                         Wireframes can be shared with stakeholders for feedback and refinement. 
                     </li>
```

Comment:
strong tags are essential to put emphasis on  texts in your website and also  more semantic way than rather using b tag, keep up the good work.